### PR TITLE
making code easier to read

### PR DESCRIPTION
**Pull request by lucming**
Signed-off-by: lucming <2876757716@qq.com>

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

**Comment by jasonliu747** [comment]
Hi @lucming thanks for your contribution. Are you interested in improving our entire logging?  For more details, you may check #210.

File: pkg/scheduler/plugins/elasticquota/plugin_helper.go [hunk 1]
```diff
@@ -82,52 +82,56 @@ func (g *Plugin) migratePods(out, in string) {
...
+		},
+	}
+	sharedWeight, _ := json.Marshal(systemElasticQuota.Spec.Max)
+	systemElasticQuota.Annotations[extension.AnnotationRuntime] = string(sharedWeight)
+	eq, err := g.client.SchedulingV1alpha1().ElasticQuotas(g.pluginArgs.QuotaGroupNamespace).
+		Create(context.TODO(), systemElasticQuota, metav1.CreateOptions{})
+	if err != nil {
+		klog.Errorf("create system group fail, err:%v", err.Error())
 	}
+	klog.V(5).Infof("create system group success, quota:%+v", eq)
```
Comment (eahydra) [inline pkg/scheduler/plugins/elasticquota/plugin_helper.go:134]: if got err, eq is nil if failed to create group

**Review by eahydra — CHANGES_REQUESTED** [review]

File: pkg/scheduler/plugins/elasticquota/plugin_helper.go (hunk 1 above)
Comment (lucming) [reply pkg/scheduler/plugins/elasticquota/plugin_helper.go:136]: > if got err, eq is nil if failed to create group

i have already update the code,thanks for review.  (^^)

File: pkg/scheduler/plugins/elasticquota/preempt.go [hunk 2]
```diff
@@ -368,11 +368,13 @@ func filterPodsWithPDBViolation(podInfos []*framework.PodInfo, pdbs []*policy.Po
...
-		resources, err := handle.ClientSet().Discovery().ServerResourcesForGroupVersion(policy.SchemeGroupVersion.String())
-		if err != nil && resources.Size() != 0 {
-			return handle.SharedInformerFactory().Policy().V1().PodDisruptionBudgets().Lister()
-		}
+	if !feature.DefaultFeatureGate.Enabled(features.PodDisruptionBudget) {
+		return nil
+	}
+
+	resources, err := handle.ClientSet().Discovery().ServerResourcesForGroupVersion(policy.SchemeGroupVersion.String())
+	if err != nil && resources.Size() != 0 {
```
[the remark this answers is not in the document]
Comment (lucming) [reply pkg/scheduler/plugins/elasticquota/preempt.go]: > if err !=nil, resources must be nil, resources.Size() will panic...

done.
this part of the code was written like this before and I didn't pay attention to it, thanks

File: pkg/scheduler/plugins/coscheduling/controller/podgroup.go [hunk 3]
```diff
@@ -200,13 +200,12 @@ func (ctrl *PodGroupController) syncHandler(key string) error {
 		}
 	}()
 	pg, err := ctrl.pgLister.PodGroups(namespace).Get(name)
-	if apierrs.IsNotFound(err) {
-		klog.Infof("PodGroup has been deleted, podGroup: %v", key)
-		return nil
-	}
 	if err != nil {
+		if apierrs.IsNotFound(err) {
+			klog.Infof("PodGroup has been deleted, podGroup: %v", key)
```
Comment (eahydra) [inline pkg/scheduler/plugins/coscheduling/controller/podgroup.go:205]: using warning maybe better

**Review by eahydra — APPROVED** [review]
/lgtm

File: pkg/scheduler/plugins/coscheduling/controller/podgroup.go (hunk 3 above)
Comment (lucming) [reply pkg/scheduler/plugins/coscheduling/controller/podgroup.go]: > using warning maybe better

done

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

**Pull request merged**